Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ChatView.swift
+//
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import SwiftUI
+import StreamChatClient
+
+@available(iOS 13, *)
+struct ChatView: View {
+    @State
+    var channelId: String
+    @State
+    var text: String = ""
+    @State
+    var messages: [String] = []
```

Comment:
lol the formatter :D Should be fixed now, right @buh ?

Reply:
This is I explicitly force to write on the same line. I no idea why it's here.
But yes it should be fixed.

Reply:
I ran the formatter manually and pushed a fix. Thought it would do it on pre-commit